glossary.md

```diff
@@ -1,21 +1,25 @@
-# Glossary
+# Glossary and abbreviations
 
 **Activation market:** Flexibility market where the assets are dispatched for the 
  period covered by a trade.  
 
-**Baseline:** The forecasted aggregated consumption/injection of an asset portfolio (see portfolio) during a defined time period
+**Asset:** In this context, an asset represents a physical or vertical device which 
+has power characteristics that can be controlled, thus providing flexibility. 
+Typically assets are either consumers (e.g. heating systems, factories) or producers
+(e.g. renewable energy sources or traditional power plants), while some assets
+(e.g. batteries) can both produce and consume. 
 
-**Block order:** Type of order for which the amount of flexibility proposed is fully indivisible (i.e. either the offer is fully accepted, either fully rejected)
+**Baseline:** The forecasted aggregated consumption/production of an asset portfolio (see portfolio) during a defined time period. 
 
-**Curtailable block orders:** type of order that can be partially accepted but above a minimum acceptance ratio defined by the seller
-
-**DSO:** Distribution System Operator
+**DSO:** Distribution System Operator, the entity responsible for operating a electrical grid. 
 
 **Fill-and-kill:** Type of order that is immediately matched. The order is killed and instantly removed after matching, whether it is matched fully, partially or not at all. Available in NODES continuous market. Commonly abbreviated as FaK. Also known as Immediate-or-Cancel (IoC).
 
 **Fill-or-kill:** Type of order that is immediately matched. The order is either matched in full and then killed, or else it is killed and instantly removed. No partial matching is allowed. Available in NODES continuous market. Commonly abbreviated as FoK. 
 
-**FSP:** Flexibility Service Provider  
+**FMO:** Flexible Market Operator, a market where participants can trade flexibility. 
```

Comment:
I would say : "Flexibility Market Operator, the operator of the platform hosting a market on which participants can trade flexibility"

Reply:
agreed